Mage/EventManager.cs

```diff
@@ -3,38 +3,53 @@
 using Wizcorp.MageSDK.Event;
 using Wizcorp.MageSDK.Log;
 
-namespace Wizcorp.MageSDK.MageClient {
-	public class EventManager : EventEmitter<JToken> {
-		private Mage mage { get { return Mage.Instance; } }
-		private Logger logger { get { return mage.logger("eventManager"); } }
+namespace Wizcorp.MageSDK.MageClient
+{
+	public class EventManager : EventEmitter<JToken>
+	{
+		private Mage Mage
+		{
+			get { return Mage.Instance; }
+		}
+
+		private Logger Logger
+		{
+			get { return Mage.Logger("eventManager"); }
```

Comment:
I grepped the whole mage sdk and seems there is inconsistency here.
Some places use upper start others dont.
I like your suggestion, I'm going to go ahead and fix this in all places 👍